tools/pydocgen/pydocgen/__init__.py

```diff
@@ -70,6 +73,23 @@ class Context(NamedTuple):
     mdoutdir: str
     input: Input
 
+
+class CreateMarkdownInput(NamedTuple):
+    """
+    The CreateMarkdownInput is the input to create_markdown_file.
+    """
+    package_name: str
+    """
```

Comment:
I actually don't know if this is how one adds a doc-comment for a property of a class?